pkg/gather/gather.go

```diff
@@ -62,7 +63,7 @@ func CreateAllGatherers(
 	return []gatherers.Interface{clusterConfigGatherer, workloadsGatherer}
 }
 
-// CollectAndRecord gathers enabled functions of the provided gatherer and records the results to the recorder
+// CollectAndRecordGatherer CollectAndRecord gathers enabled functions of the provided gatherer and records the results to the recorder
```

Comment:
`CollectAndRecordGatherer CollectAndRecord gathers`
it's somewhat duplicated....